student_explorer/cron.py

```diff
@@ -61,18 +63,17 @@ def mentor_in_athletic(self, mentor_uniqname):
             logger.warn(f"Cannot find user {mentor_uniqname} in MCommunity")
         else:
             advisor_json = json.loads(resp.text)
-            affiliation = advisor_json.get("person").get("affiliation", "")
+            user_affiliation = advisor_json.get("person").get("affiliation", "")
             # array of affiliated department.
             # The advisors of those department will be added as observers to student's Canvas courses
-            affiliated_depts = ["Athletics - Faculty and Staff"]
-            for dept in affiliated_depts:
-                if dept in affiliation:
+            for dept in deptAffiliations:
+                if dept in user_affiliation:
                     # advisor with right dept affiliation
-                    in_athletic = True
+                    in_affiliated_department = True
```

Comment:
You can just either `break` here or just return True and False at the end, to avoid going through unnecessary iterations.

Though I think python has a [one liner either using a lambda or any](https://stackoverflow.com/q/10668282/3708872) to do this better.